cmd/markdown/atmos_terraform_apply_usage.md

```diff
@@ -0,0 +1,7 @@
+Examples:
+
+  – Execute a terraform apply
+
+```
+    $ atmos terraform apply <component-name> -s <stack-name>
+```
```

Comment:
The code fence should be related to the bullet point.

```suggestion
  – Execute a terraform apply

    ```
        $ atmos terraform apply <component-name> -s <stack-name>
    ```
```

Reply:
I can do that but my observation showed it does not make any difference for the markdown.